lib/manager/common.ts

```diff
@@ -18,6 +18,7 @@ export interface ExtractConfig extends ManagerConfig {
   endpoint?: string;
   global?: any;
   gradle?: { timeout?: number };
+  helm?: { aliases?: Record<string, string> };
```

Comment:
is this the right way to add additional config?
also there's this 
 https://github.com/renovatebot/renovate/blob/84d5632bfd8a8664bb62352300d5b530724eb00c/renovate-schema.json#L1057
but the key is `helm-requirements` which doesn't play well in javascript land, will it get auto converted to `helmRequirements` or am I on the wrong track?

Reply:
You can surround it in `'helm-requirements'` anywhere necessary